jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>what is happening here now in detail? can we add some more context to what is performed in the cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>It runs MuSiC for each sample. We can add something. I don't know if we really want to dive into how MuSiC works but we can if you think it's necessary. We just don't want to focus on MuSiC so much since we can't say it's the best method.</p>

Reply:
<p>Okay now?</p>

Reply:
<p>Can you reduce the stdout a little bit? It's rather verbose.</p><p> </p><p>Anna is totally right. You need to explain a bit more what the code does that you wrote. This is NOT about explaining MuSiC but rather your code.</p>

Reply:
<p>Will add comments</p>